pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/UfoerOmregningEnslig.kt

```diff
@@ -162,24 +162,20 @@ object UfoerOmregningEnslig : AutobrevTemplate<UfoerOmregningEnsligDto> {
             ifNotNull(barnetilleggSaerkullsbarnVedVirk) { barnetilleggSaerkullsbarnVedVirk ->
 
                 val harNettoBeloep = barnetilleggSaerkullsbarnVedVirk.beloep.greaterThan(0)
-                val harJusteringsbeloepSaerkull = barnetilleggSaerkullsbarnVedVirk.justeringsbeloepAar.notEqualTo(0)
+                val barnetilleggForSaerkullsbarnVedvirk_HarjusteringsBeloepAr = barnetilleggSaerkullsbarnVedVirk.justeringsbeloepAar.notEqualTo(0)
```

Comment:
Hvorfor skal vi skrive det så verbost? det betyr egentlig at vi har justeringsbeløp for særkull.